docs/source/weights_loading.rst

```diff
@@ -137,6 +138,20 @@ You can manually save checkpoints and restore your model from the checkpointed s
     trainer.save_checkpoint("example.ckpt")
     new_model = MyModel.load_from_checkpoint(checkpoint_path="example.ckpt")
 
+Manual saving with accelerators
```

Comment:
Perhaps it will be better to just add a warning that lightning makes sure saving is done only on one process when running using accelerators, but the behaviour is not guaranteed when using loggers that don't use lightning default save, such as mlflow.

Reply:
I added a description of how to remedy the situation if you use custom loading functions. In the case of mlflow, I think this would be enough for the team to resolve their issue with saving!

Adding a warning would be nice, but a bit tricky since we would need to catch how mlflow saves the model, and override pytorch state_dicts which would be messy. I'll sync up with the mlflow team, but hopefully the additional doc change I added explains verbose enough for them